multi-agent/tests/unit/resources/test_a2a_builtin_overlay_fields.py

```
@@ -0,0 +1,63 @@
+        """base_url is the built-in resource's identity and auth_method is
+        the admin's choice of *how* callers authenticate — both must stay
+        read-only on the overlay; only the caller's own credentials
+        (sign_in / bearer_token) are user-configurable."""
+        configurable = _configurable_keys(A2AAgentNodeConfig)
+        assert ADMIN_CONTROLLED_FIELDS.isdisjoint(configurable)
+
+
+class TestA2AProviderBuiltinOverlay:
+    def test_auth_fields_are_configurable(self):
+        configurable = _configurable_keys(A2AProviderConfig)
+        assert REQUIRED_CONFIGURABLE_FIELDS <= configurable
```

Ruff (0.16.0)

[warning] Yoda condition detected

Rewrite as `configurable >= REQUIRED_CONFIGURABLE_FIELDS`

(SIM300)